### PR TITLE
Update eXXXXXXX to EXXXXXXXX and change Address to Major mentions

### LINKED ISSUE #146: Change Address to Major in Docs
This is a continuation of #107 

Non exhaustively, changes to be done in:

1. Documentation (DG and UG)
2. UML diagrams (if have mention of address)
3. Tests
4. Help

### PR DESCRIPTION
**Pull request by belligerentbeagle**
Closes #146 

I'm not 100% certain if in the intermediary-parsing steps it is using capital E as well. Would appreciate any comments or help.

Screenshot 2024-04-10 at 3 12 17 PM

### CODE REVIEW

File: docs/diagrams/ArchitectureSequenceDiagram.puml [hunk 1]
```diff
@@ -8,10 +8,10 @@ Participant ":Logic" as logic LOGIC_COLOR
 Participant ":Model" as model MODEL_COLOR
 Participant ":Storage" as storage STORAGE_COLOR
 
-user -[USER_COLOR]> ui : "delstu e1234567"
+user -[USER_COLOR]> ui : "delstu E1234567"
 activate ui UI_COLOR
 
-ui -[UI_COLOR]> logic : execute("delstu e1234567")
+ui -[UI_COLOR]> logic : execute("delstu E1234567")
```
Comment (tituschewxj) [inline docs/diagrams/ArchitectureSequenceDiagram.puml:14]:
```suggestion
ui -[UI_COLOR]> logic : execute("delstu nn/E1234567")
```

File: docs/diagrams/ArchitectureSequenceDiagram.puml [hunk 2]
```diff
@@ -8,10 +8,10 @@ Participant ":Logic" as logic LOGIC_COLOR
 Participant ":Model" as model MODEL_COLOR
 Participant ":Storage" as storage STORAGE_COLOR
 
-user -[USER_COLOR]> ui : "delstu e1234567"
+user -[USER_COLOR]> ui : "delstu E1234567"
```
Comment (tituschewxj) [inline docs/diagrams/ArchitectureSequenceDiagram.puml:11]:
```suggestion
user -[USER_COLOR]> ui : "delstu nn/E1234567"
```

File: docs/diagrams/DeleteSequenceDiagram.puml [hunk 3]
```diff
@@ -14,10 +14,10 @@ box Model MODEL_COLOR_T1
 participant "m:Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("delstu e1234567")
+[-> LogicManager : execute("delstu E1234567")
```
Comment (tituschewxj) [inline docs/diagrams/DeleteSequenceDiagram.puml:17]:
```suggestion
[-> LogicManager : execute("delstu nn/E1234567")
```

File: docs/diagrams/DeleteSequenceDiagram.puml [hunk 4]
```diff
@@ -14,10 +14,10 @@ box Model MODEL_COLOR_T1
...
 activate LogicManager
 
-LogicManager -> AddressBookParser : parseCommand("delstu e1234567")
+LogicManager -> AddressBookParser : parseCommand("delstu E1234567")
```
Comment (tituschewxj) [inline docs/diagrams/DeleteSequenceDiagram.puml:20]:
```suggestion
LogicManager -> AddressBookParser : parseCommand("delstu nn/E1234567")
```

File: docs/diagrams/UndoRedoState1.puml [hunk 5]
```diff
@@ -4,7 +4,7 @@ skinparam ClassFontColor #000000
 skinparam ClassBorderColor #000000
 skinparam ClassBackgroundColor #FFFFAA
 
-title After command "delstu e1234567"
+title After command "delstu E1234567"
```
Comment (tituschewxj) [inline docs/diagrams/UndoRedoState1.puml:7]:
```suggestion
title After command "delstu nn/E1234567"
```

**Review by tituschewxj — APPROVED** [review]
LGTM! Just a few nits, but yes, it would be great to update eXXXXXXX to EXXXXXXX in the diagrams also.

**Review by ForAeons — APPROVED** [review]
LGTM!

**Pull request merged**